test/utils/rotation/etcd_encryption_key.go

```diff
@@ -210,6 +151,12 @@ func (v *ETCDEncryptionKeyVerifier) AfterCompleted(ctx context.Context) {
 	}).Should(Succeed(), "etcd encryption config should only have new key")
 }
 
+// ExpectCompletingStatus is called while waiting for the Completing status.
```

Comment:
Can we add comments similar to https://github.com/gardener/gardener/blob/8d68bd74d497ebca589ce7f23cf9417d09606bf4/test/utils/rotation/observability.go#L101-L102 about what's happening here? It's not very obvious.